plugins/inputs/redfish/README.md

```diff
@@ -0,0 +1,165 @@
+# Redfish Input Plugin
+
+The `redfish` plugin gathers  metrics and status information about CPU temperature, fanspeed, Powersupply, voltage, hostname and Location details(datacenter,placement,rack and room) of Dell hardware servers for which redfish is enabled.
+And also metrics like CPU temperature,fanspeed, Powersupply and hostname metrics for HP Hardware server(redfish should be enabled).
+
+Note: Currently this plugin only supports DELL and HP servers.
+
+
+
+### Configuration
+
+```toml
+[[inputs.redfish]]
+## Server OOB-IP
+host = "http://192.0.0.1"
+
+## Username,Password for hardware server
+basicauthusername = "test"
+basicauthpassword = "test"
+## Server Vendor(dell or hp)
+server= "dell"
+## Resource Id for redfish APIs
+id="System.Embedded.1"
+## Optional TLS Config
+# tls_ca = "/etc/telegraf/ca.pem"
+# tls_cert = "/etc/telegraf/cert.pem"
+# tls_key = "/etc/telegraf/key.pem"
+## Use TLS but skip chain & host verification
+# insecure_skip_verify = false
+
+## Amount of time allowed to complete the HTTP request
+# timeout = "5s"
+```
+
+### Metrics for Dell Servers
+
+- cpu_temperatures
+	- tags:
+		- hostname
+		- name
+		- oob_ip
+		- host
```

Comment:
I think management IP(oob_ip ) of hardware server would be helpful as there is a possibility multiple hardware server can have similar hostname, oob_ip helps there. It would be also helpful to co-relate  hostname with oob_ip. can we rename oob_ip  to source_ip and hostname to source_hostname?

Reply:
Okay, let's call hostname -> `source` and oob_ip -> `source_ip` to match with https://github.com/influxdata/telegraf/issues/4413